chart/templates/configmap.yaml

```diff
@@ -55,3 +55,11 @@ data:
   known_hosts: |
     {{ .Values.dags.gitSync.knownHosts | nindent 4 }}
 {{- end }}
+{{- if eq .Values.executor "KubernetesExecutor" }}
+  pod_template_file.yaml: |-
+{{- if .Values.podTemplateFile }}
```

Comment:
This value name needs clarifying -- seeing the name `file` I'd expect this to be a path to the file, but if I'm reading this correctly the helm value is the contents of the file/YAML to use as the pod template, correct?

If so `.Values.podTemplate` or `.Values.executorPodTemplate` might be a better name